.github/workflows/security.yml

```diff
@@ -26,18 +26,22 @@ jobs:
         env:
           FOSSA_API_KEY: ${{ secrets.FOSSA_API_KEY }}
         run: |
+          DIFF_OPTION=""
           MAIN_BRANCH="main"
           MAIN_BRANCH_REF="refs/heads/$MAIN_BRANCH"
-          CURRENT_BRANCH_REF="${{ github.ref }}"
-          DIFF_OPTION=""
+
+          CURRENT_BRANCH_REF="${{ github.head_ref || github.ref }}"
+          echo "Current Branch: $CURRENT_BRANCH_REF"
 
           if [ "$CURRENT_BRANCH_REF" != "$MAIN_BRANCH_REF" ]; then
             git fetch origin $MAIN_BRANCH
             MAIN_SHA=$(git rev-parse origin/$MAIN_BRANCH)
             DIFF_OPTION="--diff $MAIN_SHA"
           fi
 
-          fossa test --format json $DIFF_OPTION | tee fossa-test.json
+          CMD="fossa test --format json $DIFF_OPTION"
+          echo "Running FOSSA Test with command: '$CMD'"
```

Comment:
maybe we should just use `set +x` to make it clear what command is being run?

Reply:
Good suggestion! Done!